helm/prometheus-meta-operator/templates/prometheus-rules/prometheus.rules.yml

```diff
@@ -22,3 +22,17 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    ## Pages Atlas when prometheus fails to send samples to cortex
+    - alert: PrometheusFailsToCommunicateWithRemoteStorageAPI
+      annotations:
+        description: '{{`Prometheus can''t communicate with Remote Storage API at {{ $labels.url }}.`}}'
+      expr: rate(prometheus_remote_storage_samples_failed_total{app="prometheus"}[1h]) > 0.10 or rate(prometheus_remote_storage_samples_total[1h]) == 0
```

Comment:
Why do we have the `app="prometheus"` selector on one metric, but no the other?

Reply:
I think we can just remove it